spacelift/provider.go

```diff
@@ -20,30 +20,32 @@ func Provider() terraform.ResourceProvider {
 			},
 		},
 		DataSourcesMap: map[string]*schema.Resource{
-			"spacelift_context_attachment":        dataContextAttachment(),
-			"spacelift_context":                   dataContext(),
-			"spacelift_environment_variable":      dataEnvironmentVariable(),
-			"spacelift_ips":                       dataIPs(),
-			"spacelift_mounted_file":              dataMountedFile(),
-			"spacelift_policy":                    dataPolicy(),
-			"spacelift_stack_aws_role":            dataStackAWSRole(),
-			"spacelift_stack_gcp_service_account": dataStackGCPServiceAccount(),
-			"spacelift_stack_webhook":             dataStackWebhook(),
-			"spacelift_stack":                     dataStack(),
-			"spacelift_worker_pool":               dataWorkerPool(),
+			"spacelift_aws_role":             dataAWSRole(),
+			"spacelift_context_attachment":   dataContextAttachment(),
+			"spacelift_context":              dataContext(),
+			"spacelift_environment_variable": dataEnvironmentVariable(),
+			"spacelift_gcp_service_account":  dataGCPServiceAccount(),
```

Comment:
Please make sure not to remove old resources (eg. `spacelift_stack_gcp_service_account`), just point them to new handlers which should still work.